src/components/Search/SearchResultsList.jsx

```diff
@@ -1,49 +1,40 @@
 import React from 'react';
 
 import Box from '@mui/material/Box';
-import Grid from '@mui/material/Grid';
 import List from '@mui/material/List';
-import ListItem from '@mui/material/ListItem';
-import ListItemButton from '@mui/material/ListItemButton';
-import ListItemText from '@mui/material/ListItemText';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 
+import SearchResultsListItem from './SearchResultsListItem';
+
 function SearchResultsList(props) {
-  const { searchResults } = props;
+  const { searchResults, searchIsFocused } = props;
   return (
-    <Grid container>
-      <Grid item xs={3} />
-      <Grid item xs={9}>
-        <Box
-          sx={{
-            maxWidth: '100%',
-            bgcolor: 'background.paper',
-            border: 1,
-            borderColor: 'grey.500',
-            borderRadius: '4px',
-            '&:focus': {
-              borderColor: 'primary.main',
-            },
-            '&:hover': {
-              borderColor: 'text.primary',
-            },
-          }}
-          alignItems="flex-end"
-        >
-          <List>
-            {searchResults &&
-              searchResults.map((item) => (
-                <ListItemButton key={item.id}>
-                  <ListItem disablePadding>
-                    <ListItemText primary={item.revision} />
-                  </ListItem>
-                </ListItemButton>
-              ))}
-          </List>
-        </Box>
-      </Grid>
-    </Grid>
+    searchIsFocused && (
+      <Box
+        id="search-revision-list-box"
```

Comment:
added ID for detecting when this element is clicked (or not)

Reply:
Is this id still used?

Reply:
nope!